### PR TITLE
making it so hostel will honor qa apps hostnames

### PR DESCRIPTION
**Pull request by ebressler**
making it so hostel will honor qa apps by looking for a variable we already set in qa apps on heroku. The variable is QA_HOST

### CODE REVIEW

File: lib/hostel/site.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def build_path(path, secure: true, path_args: {})
         'http://'
       end
 
-      url << domain
+      url << ENV['QA_HOST']?ENV['QA_HOST']:domain
```
Comment (zcotter) [inline lib/hostel/site.rb]: Is your space bar broken?
If not, and you are just checking if `ENV['QA_HOST']` is nil, might be easier with:
`ENV['QA_HOST'] || domain`

Comment (stacy-wells) [inline lib/hostel/site.rb]: nitpick but can you add spacing here so it is a little more readable? like this
 `ENV['QA_HOST'] ? ENV['QA_HOST'] : domain`

**Comment by jaslakson** [comment]
spaces are good, but otherwise 👍

**Comment by ebressler** [comment]
@zcotter and @stacy-wells fixed the spacing/made it more succinct

**Comment by stacy-wells** [comment]
:+1:

**Comment by zcotter** [comment]
👍

**Pull request merged**